rel/reltool.config

```diff
@@ -12,7 +12,7 @@
 
 {sys, [
     {lib_dirs, ["../src"]},
-    {rel, "couchdb", "2.1.0", [
+    {rel, "couchdb", "2.1.1", [
```

Comment:
not sure if this can be scripted. If not, we could make this the canonical source where we change the version number by hand instead of `https://github.com/apache/couchdb/blob/master/version.mk` (which I see is still on 2.1.0 too.

Reply:
Will address this later...for now I'll just fix the Makefile problem.